app/shared/components/Header/AudioPlayer/AudioPlayer.test.tsx

```diff
@@ -58,26 +61,39 @@ describe('AudioPlayer', () => {
 
   beforeEach(() => {
     jest.clearAllMocks();
-    props = {};
-    renderComponent();
+
+    mockTogglePlay = jest.fn();
+    mockPlayTrack = jest.fn();
+
+    mockContextValue = {
+      src: 'test-audio.mp3',
+      trackName: 'Test Track',
+      isPlaying: false,
+      playTrack: mockPlayTrack,
+      togglePlay: mockTogglePlay
+    };
   });
 
   test('should render AudioPlayer component', () => {
```

Comment:
Refactor in this file test cases names please too
 ```suggestion
  it('should render AudioPlayer component', () => {
```